### PR TITLE
Explain how logging can be configured in the smallrye-graphql-client guide

### PR DESCRIPTION
**Pull request by jmini**
Based on this discussion: 
https://stackoverflow.com/questions/77184741/how-to-log-http-requests-made-by-quarkus-graphql-client/

### CODE REVIEW

File: docs/src/main/asciidoc/smallrye-graphql-client.adoc [hunk 1]
```diff
@@ -256,6 +257,17 @@ With this REST endpoint included in your application, you can simply send a GET
 and the application will use an injected typesafe client instance to call the remote service, obtain
 the films and planets, and return the JSON representation of the resulting list.
 
+=== Logging
+
+For debbuging purpose, it is possible to log the request generated by the typesafe client and the response sent back by the server by changing the log level of the `io.smallrye.graphql.client` category to `TRACE` (See xref:logging.adoc#configure-the-log-level-category-and-format[Logging guide]).
```
Comment (jmartisk) [inline docs/src/main/asciidoc/smallrye-graphql-client.adoc:262]: Please fix debbuging to debugging

Otherwise, LGTM!

**Review by jmartisk — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/smallrye-graphql-client.adoc [hunk 2]
```diff
@@ -256,6 +257,17 @@ With this REST endpoint included in your application, you can simply send a GET
...
+
+This can be achieved by adding following lines to `application.properties`:
+
+```
+quarkus.log.category.\"io.smallrye.graphql.client\".level=TRACE
+quarkus.log.category.\"io.smallrye.graphql.client\".min-level=TRACE
```
Comment (gsmet) [inline docs/src/main/asciidoc/smallrye-graphql-client.adoc:268]: I'm surprised by the `\` here? I don't think they are needed?

File: docs/src/main/asciidoc/smallrye-graphql-client.adoc [hunk 3]
```diff
@@ -256,6 +257,17 @@ With this REST endpoint included in your application, you can simply send a GET
 and the application will use an injected typesafe client instance to call the remote service, obtain
 the films and planets, and return the JSON representation of the resulting list.
 
+=== Logging
+
+For debbuging purpose, it is possible to log the request generated by the typesafe client and the response sent back by the server by changing the log level of the `io.smallrye.graphql.client` category to `TRACE` (See xref:logging.adoc#configure-the-log-level-category-and-format[Logging guide]).
+
+This can be achieved by adding following lines to `application.properties`:
+
+```
```
Comment (gsmet) [inline docs/src/main/asciidoc/smallrye-graphql-client.adoc:266]: Please use a source block here with `[source,properties]`.

We had a few issues with the Markdown style when pushing to the website.

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks for this nice addition.

Please see @jmartisk and the two additional comments I added.
Nothing big really.

Thanks!

**Review by gsmet — APPROVED** [review]
I adjusted things a bit, it should be ready to go once CI is green.

**Review by jmartisk — APPROVED** [review]

**Pull request merged**